deps/Makefile

```diff
@@ -441,6 +446,9 @@ endif # ARCH == ppc64
 
 
 ifeq ($(LLVM_SANITIZE),1)
+# NOTE: LLVM_SANITIZE + USE_LLVM_SHLIB only works with LLVM_USE_CMAKE, because
```

Comment:
This is a good candidate for an error indeed (I didn't consider it because most other broken scenario's are LLVM version dependent, which is hard to check for).